requirements.txt

```diff
@@ -0,0 +1,2 @@
+numpy==2.2.0
+ruff==0.8.4
```

Comment:
Need actual content for requirements.txt to generate `/root/.local/bin` directory